packages/cli/src/commands/ui/create/react.ts

```diff
@@ -54,9 +56,16 @@ export default class React extends Command {
   async run() {
     const args = this.args;
 
-    await this.createProject(args.name);
+    const finalOutput = await this.createProject(args.name);
+
+    cli.action.start('Creating search token server');
     await this.setupServer(args.name);
     await this.setupEnvironmentVariables(args.name);
+    cli.action.stop();
+
+    console.log(EOL);
```

Comment:
I'd use the Oclif log to be safe
```suggestion
    this.log(EOL);
```